src/lib/components/Toast.svelte

```diff
@@ -0,0 +1,44 @@
+<script lang="ts">
+  import { fade } from 'svelte/transition';
+	import CloseToastIcon from './icons/CloseToastIcon.svelte';
+	import WarningIcon from './icons/WarningIcon.svelte';
+  import { createEventDispatcher } from 'svelte';
+
+  const dispatch = createEventDispatcher();
+
+</script>
+
+<div class="toast-dialog" role="alert" transition:fade>
+	<WarningIcon />
+	<div class="text">Missing required player info.</div>
```

Comment:
Let's extract this into a prop to make this more reusable
```suggestion
	<div class="text">{message}</div>
```